java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -283,6 +283,37 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<ClusterValue<String[]>> time(Route route);
 
+    /**
+     * Synchronously saves the database(s) to disk.<br>
```

Comment:
Nit:
Performs a synchronous save of the dataset producing a point in time snapshot of all the data inside the Redis instance, in the form of an RDB file.
Just copy valkey/redis? Could be confusing to see the keyword snapshot in the return but not talked about in the description.

Reply:
Copied from there:
```
127.0.0.1:6379> command docs save
1# "save" =>
   1# "summary" => "Synchronously saves the database(s) to disk."
   2# "since" => "1.0.0"
   3# "group" => "server"
   4# "complexity" => "O(N) where N is the total number of keys in all databases"
```